docs/developer/app-store/apps/cms.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: CMS
+sidebar_position: 4
+---
+
+# CMS
+
+:::note V2 version
+This document describes the v2 version of the app. The v1 version is deprecated and will be removed in the future.
+:::
+
+## Overview
+
+_CMS_ is a Saleor app that allows one-direction synchronization of Saleor products into supported CMS platforms. Here are the currently available platforms:
+
+- [Contentful](https://www.contentful.com/)
+- [Strapi](https://strapi.io/)
+- [DatoCMS](https://www.datocms.com/)
+
+:::note Missing integrations?
+We listen to the community's voice. Let us know if you miss any integration by [filling out a survey](https://saleor.typeform.com/to/ecR2msLk?typeform-source=apps.staging.saleor.io)
+:::
+
+The app is flexible and allows you to configure multiple configurations of multiple providers and assign them per channel.
+
+:::caution
+
+To successfully configure the CMMS App, you must have an account with one of the supported CMS platforms.
+
+:::
+
+## Main assumptions
+
+- The app is unidirectional. It means it will only send product updates from Saleor to the CMS platform. If you change data in the CMS, it will not be reflected in Saleor.
+- The app considers [Product Variant](https://docs.saleor.io/docs/3.x/api-reference/products/objects/product-variant) as a main entity that is synchronized. Every created item in the CMS will be single, unique variant of the product.
+- The app requires you to create a model in the CMS platform, but you can map Saleor product fields to the CMS fields.
+- The app will synchronize product variants on the following events:
```

Comment:
**suggestion (non-blocking)**: link to asynchronous events documentation to provide context about what type of events we mean.

```suggestion
- The app will synchronize product variants on the following [events](https://docs.saleor.io/docs/3.x/developer/extending/apps/asynchronous-webhooks):
```